apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
             user = NULL;
+            if (mode == OPT_LIST)
+                break;
```

Comment:
The if was to follow the logic of the while condition, but if required I can make it unconditional, it would not change anything functionally.